### DISCUSSION TITLE
Salesforce Authentication Issue

### DISCUSSION THREAD

**Discussion by anilacrosswork** [Authenticator-Extension/Authenticator#1155] [Q&A]
I have an end user who is trying to use this to authenticate with Salesforce. Everytime she tries to use the code, it has already expired. I am the Salesforce Admin and I haven't had this issue before. Unsure how to get it back up and running for her.

**Comment by Sneezry** [comment]
Correcting the clock on her machine should work.

**Comment by anilacrosswork** [comment]
> Correcting the clock on her machine should work.
The time seems to be correct, any other suggestions?
